installer/tue-install-impl.bash

```diff
@@ -1483,6 +1478,7 @@ then
     tue-install-snap-now "$TUE_INSTALL_SNAPS"
 fi
 
+
```

Comment:
Not needed IMO

Reply:
Its in line with all the double line-endings surrounding the functions above :wink: 
It is a diff with our repo, not sure where it came from.